packages/firebase/index.d.ts

```diff
@@ -1220,6 +1234,39 @@ declare namespace firebase.app {
   }
 }
 
+/**
+ * @webonly
+ */
+declare namespace firebase.installations {
+  /**
+   * The Firebase Installations service interface.
+   *
+   * Do not call this constructor directly. Instead, use
+   * {@link firebase.installations `firebase.installations()`}.
+   */
+  export interface Installations {
+    /**
+     * Creates a Firebase Installation if there isn't one for the app and
+     * returns the Installation ID.
+     *
+     * @return Firebase Installation ID
+     */
+    getId(): Promise<string>;
```

Comment:
Why is this not called #getFid()?

In the context of Firebase Installations, ID might be clear, but still there are so many IDs (Android-IDs, Gaia-IDs, Project-IDs, Instance-IDs, and so on and so forth) that I would like to have a distinctive name.